README.md

```diff
@@ -1,4 +1,4 @@
-Bitcoin Core integration/staging tree
```

Comment:
![Screenshot 2020-12-04 at 19 49 30](https://user-images.githubusercontent.com/3534524/101208327-ee280c00-3669-11eb-9405-038a2e5dc7b9.png)

IMHO not worth it.